src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -841,20 +866,23 @@ void handle_actor_task_submitted(LocalSchedulerState *state,
      * will be resubmitted (internally by the local scheduler) whenever a new
      * actor notification arrives. */
     utarray_push_back(algorithm_state->cached_submitted_actor_tasks, &spec);
+    utarray_push_back(algorithm_state->cached_submitted_actor_task_sizes,
```

Comment:
We only have access to the size once the flatbuffer builder has been finalized, so we can't store it in the message itself.